src/bepasty/config.py

```diff
@@ -1,136 +1,133 @@
+    'HIGHLIGHT_TYPES': 100 * 1000,
+    # the in-browser pdf reader is sometimes rather slow and should
+    # rather not be used for big PDFs:
+    'application/pdf': 10 * 1000 * 1000,
+    'application/x-pdf': 10 * 1000 * 1000,
+    # images / audio / video can be rather big, we do not process them:
+    'image/': 10 * 1000 * 1000,
+    'audio/': 1 * 1000 * 1000 * 1000,
+    'video/': 5 * 1000 * 1000 * 1000,
+    # DEFAULT - matches everything not matched otherwise.
+    # As we have catched everything we are able to render already,
+    # this maybe should be a medium size, just for the case we forget
+    # something:
+    '': 1 * 1000 * 1000,
+}
+
+# Whether to use the python-magic module to guess a file's mime
+# type by looking into its content (if the mime type can not be
+# determined from the filename extension).
+# NOTE:
+# libmagic may have security issues, so maybe you should only use
+# it if you trust all users with upload permissions ('create').
+USE_PYTHON_MAGIC = False
+
+#: Define storage backend, choose from:
+#:
+#: - 'filesystem'
+#:
+STORAGE = 'filesystem'
+
+#: Filesystem storage path
+STORAGE_FILESYSTEM_DIRECTORY = '/tmp/'
+
+#: server secret key needed for safe session cookies.
+#: you must set a very long, very random, very secret string here,
+#: otherwise bepasty will not work (and crash when trying to log in)!
+SECRET_KEY = ''
+
+#: transmit cookie only over https (if you use http, set this to False)
+SESSION_COOKIE_SECURE = True
+#: use a permanent session (True, cookie will expire after the given
+#: time, see below) or not (False, cookie will get removed when browser
+#: is closed)
+PERMANENT_SESSION = False
+#: lifetime of the permanent session (in seconds)
+PERMANENT_SESSION_LIFETIME = 31 * 24 * 3600
+
+#: Bepasty does **not** use the usual user/password scheme, but **only**
+#: uses passwords (or passphrases - we'll call both "a secret" below) as
+#: log-in credentials - there are no separate user names.
+#:
+#: People who log-in using such a secret may get more permissions than
+#: those who do not log-in (who just get DEFAULT_PERMISSIONS).
+#:
+#: Depending on the secret used to log-in, they will get the permissions
+#: configured here, see below.
+#:
+#: You can use same secret / same permissions for all privileged users or
+#: set up different secrets / different permissions for each user.
+#:
+#: If you want to be able to revoke permissions for some user / some group
+#: of users, it might be a good idea to remember to whom you gave which
+#: secret (and also handle it in a rather fine-grained way).
+#:
+#: PERMISSIONS is a dict that maps secrets to permissions, use it like:
+#:
+#: ::
+#:
+#:     PERMISSIONS = {
+#:         'myadminsecret_1.21d-3!wdar34': 'admin,list,create,modify,read,delete',
+#:         'uploadersecret_rtghtrbrrrfsd': 'create,read',
+#:         'joe_doe_89359299887711335537': 'create,read,delete',
+#:     }
+PERMISSIONS = {
+    # 'foo': 'admin,list,create,modify,read,delete'
```

Comment:
please keep the line as it was, the comma at the end was intended. Just indent it correctly to match your other changes.

Reply:
Done :). Neat trick with the ` ?w=1`. I couldn't figure it out at the beginning but I found a button in the GitHub UI page which hinted me to its functionality.